external-crates/move/crates/move-ir-to-bytecode/src/compiler.rs

```diff
@@ -47,6 +47,11 @@ macro_rules! record_src_loc {
             .source_map
             .add_parameter_mapping($context.current_function_definition_index(), source_name)?;
     }};
+    (_return: $context:expr, $_type:expr) => {{
```

Comment:
Nit: I'd probably use `return_` here. We usually have the convention of "suffix `_` to shadow a keyword" and "prefix `_` to mean an unused or ignored variable".